gridfs/grid_file.py

```diff
@@ -655,7 +655,7 @@ def seekable(self) -> bool:
         return True
 
     def __iter__(self) -> "GridOut":
-        """Return an iterator over all of this file's data.
+        r"""Return an iterator over all of this file's data.
```

Comment:
Please remove the changes to this file. I already fixed this last September in PYTHON-2046. In the future please make sure you branch from the latest commit. For example:
```
git checkout master
git pull upstream master
git push
git checkout -b PYTHON-XXXX
```